charts/kube-state-metrics/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+name: kube-state-metrics
+description: Install kube-state-metrics to generate and expose cluster-level metrics
+keywords:
+- metric
+- monitoring
+- prometheus
+- kubernetes
+version: 2.9.2
```

Comment:
I suggest to bump version to 2.10.0. That allows to use 2.9.2 in the old repo to deprecate the chart over there.

Reply:
@nestorsalceda 👋 Good question, I had to think about this for a moment.

Re semver, this chart repo move isn't a backwards incompatible change (MAJOR), or a new feature (MINOR), so PATCH is fine.

In general when charts move repos they don't bump two PATCH versions to address a conflict with the deprecation bump in `stable`. We could do that here, but it give us anything – there is no version conflict since the release will be from a different repo.

If you see a problem with this line of thinking lmk. But I wouldn't think so since this is the way every other chart repo move has done it.